src/components/NewRuntimeModal.js

```diff
@@ -751,51 +773,138 @@ export class NewRuntimeModalBase extends Component {
       div(['Version: ', version || null])
     ])
 
-    const renderRuntimeSection = () => {
-      const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '0.8rem', alignItems: 'center' }
+    const renderCloudComputeProfileSection = () => {
+      const { hasGpu, gpuEnabled, gpuType, numGpus } = this.state
+      const { cpu: currentNumCpus, memory: currentMemory } = findMachineType(mainMachineType)
+      const validGpuOptions = getValidGpuTypes(currentNumCpus, currentMemory)
+      const validGpuNames = _.flow(_.map('name'), _.uniq, _.sortBy('price'))(validGpuOptions)
+      const validGpuName = _.includes(displayNameForGpuType(gpuType), validGpuNames) ? displayNameForGpuType(gpuType) : _.head(validGpuNames)
+      const validNumGpusOptions = _.flow(_.filter({ name: validGpuName }), _.map('numGpus'))(validGpuOptions)
+      const validNumGpus = _.includes(numGpus, validNumGpusOptions) ? numGpus : _.head(validNumGpusOptions)
+      const gridStyle = { display: 'grid', gridGap: '1.5rem', alignItems: 'center', marginTop: '0.75rem' }
```

Comment:
One-`gridStyle`-fit-all wasn't easy to make work since GPU selections have different widths than CPU selections, so I moved `gridTemplateColumns`down to lower-level `div`s for more control.